docs/select-query-builder.md

```diff
@@ -274,6 +274,19 @@ is a shortcut for:
 .setParameter("name", "Timber")
 ```
 
+Note: a set of parameters is unique per-query. A parameter's value will be overriden if you set it multiple times:
```

Comment:
Thank you for your contribution! But I think we just need a simple one statement, like "note: do not use same parameter names for different values across your query builder" without code examples to simplify docs.

Reply:
Updated!